.github/workflows/release-plz.yml

```diff
@@ -29,8 +29,33 @@ jobs:
       - name: Install Rust toolchain
         uses: dtolnay/rust-toolchain@stable
 
+      - name: Install wasm-pack
+        uses: jetli/wasm-pack-action@v0.4.0
+
       - name: Run release-plz
         uses: MarcoIeni/release-plz-action@v0.5
         env:
           GITHUB_TOKEN: ${{ secrets.RELEASE_PLZ_TOKEN }}
           CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+      - name: Release to npm
```

Comment:
I ended up not using release plz outputs. If something fails during npm release, release-plz won't re-trigger npm release and this version will be stuck without release. With manual version checks between npmjs and crates.io, this is somewhat idempotent.